src/test/java/org/minions/devfund/reinaldo/movie/MovieDatabaseTest.java

```diff
@@ -0,0 +1,144 @@
+package org.minions.devfund.reinaldo.movie;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by reinaldo on 26-07-18.
+ */
+public class MovieDatabaseTest {
+
+    private static final int FIRST_MOVIE = 0;
+    private static final double PARAMETER_FOR_DOUBLE = 0;
+
+    /**
+     * Get best actor in the collection.
+     */
+    @Test
+    public void testGetBestActor() {
+        final MovieDatabase movieDatabase = new MovieDatabase();
+
+        final String movie1 = "Monster";
+        final String[] movie1WithActors = {"Denzel Washington"};
+        final int movie1WithRating = 20;
+
+        final String movie2 = "The Pianist";
+        final String[] movie2WithActors = new String[]{"Adrien Brody", "Chris Pine"};
+        final int movie2WithRating = 5;
+
+        final String expectedBestActor = "Denzel Washington";
+        movieDatabase.addMovie(movie1, movie1WithActors);
+        movieDatabase.addMovie(movie2, movie2WithActors);
+        movieDatabase.updateRating(movie1, movie1WithRating);
+        movieDatabase.updateRating(movie2, movie2WithRating);
+        assertEquals(expectedBestActor, movieDatabase.getBestActor());
+    }
+
+    /**
+     * Get best movie in the collection.
+     */
+    @Test
+    public void testGetBestMovie() {
+        final MovieDatabase movieDatabase = new MovieDatabase();
+
+        final String movie1 = "Monster";
+        final String[] movie1WithActors = {"Denzel Washington", "Charlize Theron"};
+        final int movie1WithRating = 20;
+
+        final String movie2 = "The Pianist";
+        final String[] movie2WithActors = new String[]{"Adrien Brody", "Chris Pine"};
+        final int movie2WithRating = 5;
+
+        final String expectedBestMovie = "Monster";
+
+        movieDatabase.addMovie(movie1, movie1WithActors);
+        movieDatabase.addMovie(movie2, movie2WithActors);
+        movieDatabase.updateRating(movie1, movie1WithRating);
+        movieDatabase.updateRating(movie2, movie2WithRating);
+        assertEquals(expectedBestMovie, movieDatabase.getBestMovie());
+    }
+
+    /**
+     * Add new movie and actor in the collection.
+     */
+    @Test
+    public void testAddMovieActorExist() {
+        final MovieDatabase movieDatabase = new MovieDatabase();
+        final String movieOld = "The Pianist2";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "The Pianist2" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)